package.json

```diff
@@ -133,7 +132,7 @@
         "electron-updater": "4.0.14",
         "fs-extra": "8.1.0",
         "mustache": "3.0.1",
-        "nrf-device-setup": "0.6.3",
+        "nrf-device-setup": "0.6.5",
```

Comment:
Isn't it 0.6.6?

Reply:
You are right, that is better. I pushed a new commit with that. It invalides the review though, so this PR needs a new approval.